cmake/modules/BasicConfig.cmake

```diff
@@ -76,7 +83,14 @@ if (${META_PROJECT_VARNAME_UPPER}_CONFIGURATION_TARGET_SUFFIX STREQUAL "none")
 elseif (${META_PROJECT_VARNAME_UPPER}_CONFIGURATION_TARGET_SUFFIX)
     set(TARGET_SUFFIX "-${${META_PROJECT_VARNAME_UPPER}_CONFIGURATION_TARGET_SUFFIX}")
 elseif (CONFIGURATION_TARGET_SUFFIX)
-    set(TARGET_SUFFIX "-${CONFIGURATION_TARGET_SUFFIX}")
+    set(TARGET_SUFFIX "-${CONFIGURATION_TARGET_SUFFIX}-")
```

Comment:
This line looks wrong. The suffix shouldn't end with a dash.

Reply:
Please ignore this push. I somehow messed up.